TheaterSchedule/Controllers/CreateGoogleFormController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Newtonsoft.Json;
+using Microsoft.AspNetCore.Mvc;
+using TheaterSchedule.BLL.Interfaces;
+using System;
+using Microsoft.AspNetCore.Http;
+using System.Threading.Tasks;
+
+// For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace TheaterSchedule.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CreateGoogleFormController : Controller
+    {
+        private IGetDataFromGoogleFormService getService;
+        private ISendDataToGoogleFormService sendService;
+
+        public CreateGoogleFormController(IGetDataFromGoogleFormService googleFormService, ISendDataToGoogleFormService sendFormService)
+        {
+            getService = googleFormService;
+            sendService = sendFormService;
+        }
+
+        [HttpGet]
+        [Produces("application/json")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(Exception), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public ActionResult<string> GetDataFromGoogleForm(string url = "https://docs.google.com/forms/d/e/1FAIpQLSfwgTnNsb7SvCmrkskJZINvhuGY861iNfdbMZ_1UcNylORT6A/viewform")
+        {
+            try

```

Comment:
no need in try catch. Project has exception handling middleware
https://github.com/net375/theater-schedule-backend/blob/0fd0ecfd7b5f19909684672557da6041eb95a89e/TheaterSchedule/MiddlewareComponents/CustomExceptionMiddleware.cs